src/Arcus.Templates.Tests.Integration/Fixture/InMemorySecretProvider.cs

```diff
@@ -11,7 +11,7 @@ namespace Arcus.Templates.Tests.Integration.Fixture
     /// <summary>
     /// <see cref="ISecretProvider"/> implementation that provides an in-memory storage of secrets by name.
     /// </summary>
-    public class InMemorySecretProvider : ICachedSecretProvider
+    public class InMemorySecretProvider : ICachedSecretProvider, ISyncSecretProvider
```

Comment:
If I'm not mistaken, we have this `InMemorySecretProvider` in the Arcus.Testing assembly, so can't we re-use the type from there instead of duplicating it here ?